web_widget_one2many_product_picker/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Tecnativa - Alexandre D. Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Web Widget One2Many Product Picker',
+    'summary': 'Widget to select products on one2many fields',
+    'version': '12.0.1.0.0',
+    'category': 'Website',
+    'author': "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    'website': 'https://www.tecnativa.com',
+    'license': 'AGPL-3',
```

Comment:
```suggestion
    'license': 'AGPL-3',
    "depends": ["sale"],
```

Reply:
It mustn't depend on `sale`. Why is this needed?

Reply:
Ok, but a dependency must be set, isn't it?

Reply:
Yes, it should be `web`. If no dependency is set, it's assumed that is `base`.